bin/oref0-setup.sh

```diff
@@ -70,6 +75,10 @@ case $i in
     ;;
 esac
 done
+if [[ -z $TTY && radio_locale != "US" ]]; then
+    echo "World wide radio locale is only supported with TI stick currently"
```

Comment:
Aren't there WW Carelink sticks for WW pumps? If so, they should Just Work, and no further code changes should be required to support WW pumps with Carelink...